how-to/how-to-use-kata-containers-with-acrn.md

```diff
@@ -0,0 +1,111 @@
+- Optimized for a variety of IoT (Internet of Things) and embedded device solutions.
+
+Please refer ACRN [documentation](https://projectacrn.github.io/latest/index.html) for more details on ACRN hypervisor and device model.
+
+## Pre-requisites
+
+This document requires the presence of the ACRN hypervisor and Kata containers on your system, that can be installed by following:
+
+- ACRN supported [Hardware](https://projectacrn.github.io/latest/hardware.html#supported-hardware).
+- ACRN [software](https://projectacrn.github.io/latest/getting-started/apl-nuc.html#use-the-script-to-set-up-acrn-automatically) setup.
+- Kata Containers installation: Automated installation doesn't seem to be supported for Clear Linux, so please use [manual installation](https://github.com/kata-containers/documentation/blob/master/Developer-Guide.md) steps.
+
+> **Note:** Create rootfs image and not initrd image.
+
+In order to run Kata with ACRN, your container stack must provide block-based storage, such as device-mapper.
+
+> **Note:** Currently only one VM can be launched from Kata Containers using ACRN hypervisor (SDC scenario) due to [this issue](https://github.com/kata-containers/runtime/issues/1785).
+
+## Install and Configure Docker
+
+Install Docker 18.06 (as Docker 18.09 does not support device-mapper). To configure Docker for device-mapper and Kata,
+
+1. Stop Docker daemon if it is already running.
+
+```bash
+$ sudo systemctl stop docker
+```
+
+2. Set `/etc/docker/daemon.json` with the following contents.
+
+```bash
+{
+  "storage-driver": "devicemapper"
+}
+```
+
+3. Restart docker.
+
+```bash
+$ sudo systemctl daemon-reload
+$ sudo systemctl restart docker
+```
+
+4. Configure [Docker](https://github.com/kata-containers/documentation/blob/master/Developer-Guide.md#update-the-docker-systemd-unit-file) to use `kata-runtime`.
+
+## Configure Kata Containers with ACRN
+
+To configure Kata Containers with ACRN, all that is needed is to copy the configuration-acrn.toml file that is generated when building the kata-runtime to either `/etc/kata-containers/configuration.toml` or `/usr/share/defaults/kata-containers/configuration.toml`.
+
+Below command shows full paths to the configuration.toml files that the runtime loads.  The first path that exists is used. (Please make sure the kernel and image paths are set correctly in the configuration.toml file)
+
+```bash
+$ sudo kata-runtime --kata-show-default-config-paths
+```
+
+>**Warning:** Please offline CPUs using [this](offline_cpu.sh) script, else VM launches will fail.
+
+```bash
+$ sudo ./offline_cpu.sh
+```
+
+Start an ACRN based Kata Container,
+
+```bash
+$ sudo docker run -ti --runtime=kata-runtime busybox sh
+```
+
+You'll see ACRN is now running on your system, as well as a `kata-shim`, `kata-proxy`. (You should obtain an interactive shell prompt; when exiting the container, make sure that all the Kata processes are terminated
+
+```bash
```

Comment:
TL;DR - this needs to be ` ```sh `.

Detail: We try to avoid showing command *output* (as documented [here](https://github.com/kata-containers/documentation/blob/master/Documentation-Requirements.md#code-blocks)). Again, if you must show the output, you cannot use a `bash` block, but you can use a `sh` block.

Reply:
Removed the output and retained bash.